src/DocBlock/Generator.php

```diff
@@ -56,8 +56,12 @@ public function addParams(Node\Stmt\ClassMethod $node, $text)
 
         foreach ($node->getParams() as $param) {
             if ($param->type && $param->type instanceof Node\Name) {
```

Comment:
Haven't seen it earlier, but I also made a little refactor, have a look at a new PR :)